package.json

```diff
@@ -3,16 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^27.5.2",
     "@types/node": "^16.18.7",
```

Comment:
why did you install the node types ? and if needed why are they in the list of dependencies ?

Reply:
when creating CRA i used -template typescript , i think this is the default dependencies for it .